Snowflake.Data.Common/Snowflake.Data.Common.csproj

```diff
@@ -0,0 +1,78 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net471;net472</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">net6.0</TargetFrameworks>
+    <Title>Snowflake.Data.Common</Title>

```

Comment:
It seems to me that "common" has some meaning related to the underlying tools or utilities common for some other solution.
My thinking is we should name it as Snowflake.Data.Core?
WDYT? @sfc-gh-knozderko  @sfc-gh-jmartinez @sfc-gh-erojaslizano

Reply:
Core has the problem that it can suggest it to be for .NET core and not .net framework.

Reply:
@sfc-gh-mhofman We have Snowflake.Data.Core was previously the first option, and the concern of a misunderstanding that the package is only for .Net Core made us change the name, but maybe it is not a problem. I agree to use Snowflake.Data.Core